VAMobile/src/store/slices/appointmentsSlice.ts

```diff
@@ -327,17 +327,17 @@ export const getAppointmentsInDateRange =
  * Redux action to cancel appointment associated with the given cancelID
  */
 export const cancelAppointment =
-  (cancelID?: string, appointmentID?: string, isPendingAppointment?: boolean): AppThunk =>
+  (cancelID?: string, appointmentID?: string, isPendingAppointment?: boolean, status?: string, type?: string, days_to_apt?: number): AppThunk =>
```

Comment:
Super Nit, but I noticed the last param `days_to_apt` isn't camel case in a few places in this file